lib/http_ntlm.c

```diff
@@ -55,12 +53,6 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
-#if DEBUG_ME
-# define DEBUG_OUT(x) x
-#else
-# define DEBUG_OUT(x) Curl_nop_stmt
-#endif
```

Comment:
It'd be ideal to replace this with TRC logging, esp. in `vauth/ntlm.c` guard some log output behind its own `DEBUG_ME` macro.